docs/vector_store.ipynb

```diff
@@ -559,7 +556,149 @@
     "await store.aadd_texts(all_texts, metadatas=metadatas, ids=ids)\n",
     "\n",
     "# Use filter on search\n",
-    "docs = await custom_store.asimilarity_search_by_vector(query_vector, filter=\"len >= 6\")\n",
+    "docs = await custom_store.asimilarity_search(query, filter=\"len >= 6\")\n",
+    "\n",
+    "print(docs)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Create a Vector Store using existing table\n",
+    "\n",
+    "A Vector Store can be built up on an existing table.\n",
+    "\n",
+    "Assuming there's a pre-existing table in Alloy DB: `products`, which stores product details for an eComm venture.\n",
+    "\n",
+    "<details>\n",
+    "  <summary>Click for Table Schema Details</summary>\n",
+    "  \n",
+    "  ### SQL query for table creation\n",
+    "  ```\n",
+    "    CREATE TABLE products (\n",
+    "        product_id SERIAL PRIMARY KEY,\n",
+    "        name VARCHAR(255) NOT NULL,\n",
+    "        description TEXT,\n",
+    "        price_usd DECIMAL(10, 2) NOT NULL,\n",
+    "        category VARCHAR(255),\n",
+    "        quantity INT DEFAULT 0,\n",
+    "        sku VARCHAR(255) UNIQUE NOT NULL,\n",
+    "        image_url VARCHAR(255),\n",
+    "        metadata JSON,\n",
+    "        embed vector(768) DEFAULT NULL --> vector dimensions depends on the embedding model\n",
```

Comment:
I can make the dimensions lesser to make it look neater.

`textembedding-gecko@latest` produces a 768 size embedding, ignoring the dimensions param.

To reduce the vector size, a new embedding service would be needed in the notebook which might create some confusion:

```
from langchain_google_vertexai import VertexAIEmbeddings

embedding = VertexAIEmbeddings(
    model_name="text-embedding-004", project=PROJECT_ID
)
# following produces a 2 size embedding vector
embedding.embed(texts = ["sample"], dimensions = 2) // gecko ignores dimensions
```

So, for brevity, table creations schema is in a collapsed section by default and we are using a 768 size embedding